templates/template_data.go

```diff
@@ -26,6 +26,13 @@ type Data = template.Data
 
 var FromGlobs = template.FromGlobs
 
+type TemplateDefinition struct {
+	// Name of the template. Used to identify the template in the UI and when testing.
+	Name string
+	// Template string that contains the template definition.
```

Comment:
There is kind of a recursive definition going on here between the name of the struct and the comment. 

```suggestion
	// Template string that contains the template text.
```